mediachain/datastore/rocks.py

```diff
@@ -0,0 +1,51 @@
+import rocksdb
+import cbor
+from multihash import encode as multihash_encode, SHA2_256
+from base58 import b58encode
+
+
+class RocksDatastore(object):
+    def __init__(self, db_path):
+        opts = rocksdb.Options(create_if_missing=True)
+        self.db = rocksdb.DB(db_path, opts)
+
+    def get(self, ref):
+        try:
+            ref = ref.multihash_base58()
+        except AttributeError:
+            pass
+
+        ref = bytes(ref)
+        byte_string = self.db.get(ref)
+        try:
+            return cbor.loads(byte_string)
+        except ValueError:
+            return byte_string
+
+    def put(self, data_object, ref=None):
```

Comment:
yeah like it doesn't make sense to "put" to a specific key b/c it's content-addressed